HTML Editor: update AI Text Editing demo description (#30107)
DevExpress/DevExtreme: apps/demos/Demos/HtmlEditor/AITextEditing/description.md

> HTML Editor offers AI-powered text editing. Users can apply AI-powered requests to text directly within the editor's UI and paste the generated results. To enable this feature, follow these steps:

Reviewer: ```suggestion
The HTML Editor supports AI-powered text editing. Users can apply AI requests and insert the results directly into the editor. To enable this feature, follow these steps:
```

- Change style
- Change tone
- Translate
- Ask AI assistant (allows users to run their own prompts to alter HTML Editor content)

> You can also specify which predefined commands you want to include in the `"ai"` item and customize an individual standard command's default options (for example, by setting a custom list of target languages for translation). Additionally, you can add a new custom command to the `"ai"` item by specifying your own prompt. In this demo, the custom command is "Extract Keywords". For more details, check the prompt in the code.

Reviewer: ```suggestion
You can also specify which predefined commands to include in the `"ai"` item and customize an individual standard command's default options (for example, by setting a custom list of target languages for translation). Additionally, you can add a new custom command to the `"ai"` item by specifying your prompt. In this demo, the custom command is "Extract Keywords". For more details, check the prompt in the code.
```
